memphis/memphis.py

```diff
@@ -255,19 +276,28 @@ def __init__(self, connection, producer_name, station_name):
         self.producer_name = producer_name
         self.station_name = station_name
 
-    async def produce(self, message, ack_wait_sec=15):
+    async def produce(self, message, ack_wait_sec=15, headers=[]):
         """Produces a message into a station.
         Args:
             message (Uint8Array): message to send into the station.
             ack_wait_sec (int, optional): max time in seconds to wait for an ack from memphis. Defaults to 15.
+            headers ([], optional): headers that the user send with the mwssage. Defaults to []
         Raises:
             Exception: _description_
             Exception: _description_
         """
         try:
+            memphis_headers={"Nats-Msg-Id": str(uuid.uuid4()), 
+            "$memphis_producedBy": self.producer_name, 
+            "$memphis_connectionId": self.connection.connection_id}
+
+            for header in headers:
```

Comment:
there is something I want you to change here and also in the Go SDK.
instead running a for loop inside the produce function, change a little bit the Headers class/struct so that it will have an internal field in the structure you can send to the NATS api.
Talk to me in case it is not understood